certchain/certmanager/acmeclient/client.go

```diff
@@ -187,8 +197,20 @@ func NewClient(config Config) (*Client, error) {
 	}, nil
 }
 
-// Fetch acquires a new RawChain from the ACME server.
+// Fetch acquires a new RawChain from the ACME server if the current cert is
```

Comment:
s/the current cert/chain/ (or s/the current cert/the provided current certificate/)

Reply:
Done.